frontend/schemaCMS/src/routes/dataSource/dataWranglingScripts/dataWranglingScripts.component.js

```diff
@@ -120,6 +112,21 @@ export class DataWranglingScripts extends PureComponent {
     }
   }
 
+  componentDidUpdate(prevProps) {
+    const { dataWranglingScripts, dataSource } = this.props;
+
+    if (prevProps.dataWranglingScripts !== dataWranglingScripts) {
```

Comment:
better is to use equals from the ramda here if those things are not simple elements

Reply:
![image](https://user-images.githubusercontent.com/11027768/123241600-9526a500-d4e1-11eb-9468-55c70dd24232.png)

Reply:
Yes, this is intended. I want to enter condition whenever reference of this props has changed (it's related to re-fetching scripts data when you entering this particular view).